src/librustc/hir/map/mod.rs

```diff
@@ -196,9 +196,20 @@ pub struct Map<'ast> {
     map: RefCell<Vec<MapEntry<'ast>>>,
 
     definitions: RefCell<Definitions>,
+
+    num_noninlined_node_ids: usize,
```

Comment:
This should conflict with `local_node_id_watermark` from the recently merged #35114. We should make sure to merge this intelligently.